app/views/devices_guidance/4g_user_guidance.md

```diff
@@ -0,0 +1,71 @@
+
+3. Insert the battery and replace the back cover
+
+    ![The router front cover and the back cover with the battery inserted](/devices/4g4.jpg)
+
+4. Turn the router on, using the button on the top-left
+
+    ![The router with the power button on the side](/devices/4g5.jpg)
+
+5.  The lights will change colour while it is booting up, they’ll go from green to red and back to green – then the router is ready to use
+
+    ![The router powered up, with the network and power buttons lit green](/devices/4g6.jpg)
+
+6.  On your laptop or tablet, select the Wifi network – this is the SSID you noted earlier and click Connect
+7.  Tick the box that says “Connect automatically” so that your device will always connect to the router when it is turned on
+
+    ![Screenshot of Microsoft Windows showing a list of Wifi networks](/devices/4g7.png)
+
+8.  Enter the Wifi Key you noted down earlier and click Next
+
+    ![Screenshot of Microsoft Windows showing a list of Wifi networks, with the password filled in](/devices/4g8.png)
+
+9.  You’re now online
+
+## Charging your router
+
+Connect the micro-USB charging cable included into the micro-USB port on the bottom of the device and connect the other end to any USB charger.
+
+## Getting support with your router
+
+If you have any problems with your router, you should contact \[insert the contact details of your support person below\]:
+
+## Managing your monthly data allowance
+
+Your 4G router has a monthly data cap. When the router runs out of data it will not connect to the internet until more data is added at the beginning of the next month.
+
+To help prevent this, make sure your 4G router’s data connection is only used for education and for social care services.
+
+## Staying safe online
+
+Your 4G wireless router uses a filtering system that blocks some inappropriate content and limits searching to the ‘Safe Search’ supported by popular search engines. The router is also set up to comply with privacy and human rights legislation.
+
+If you’re blocked from viewing something you think you need for your work or wellbeing, get in touch with your school or local authority contact.
+
+If you’re worried that your child is accessing harmful content on any device, visit this web page to see support and advice: [https://www.gov.uk/guidance/safeguarding-and-remote-education-during-coronavirus-covid-19#communicating-with-parents-carers-and-pupils](https://www.gov.uk/guidance/safeguarding-and-remote-education-during-coronavirus-covid-19#communicating-with-parents-carers-and-pupils).
+
+If a child is worried or needs support, they can call Childline (0800 1111), download their ‘For Me’ app or visit the website: [https://www.childline.org.uk/](https://www.childline.org.uk/).
+
+[If your child has a social worker, you should contact your local authority if you have any concerns about your child’s safety or wellbeing.](https://www.childline.org.uk/)
```

Comment:
Should all of this be a link? Doesn't seem right – maybe the original was broken?